### PR TITLE
fix formatting + correct instructions re: song_notes

### PR DESCRIPTION
**Pull request by gj**
- add Markdown code formatting
- escape underscores from turning into `<em>` tags
- remove dupe Learn link

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,43 +3,40 @@
 ## Objectives
 
 1. Populate select options based on association options.
-2. Assign a FK based on a select box value directly through mass assignment. (post[category_id])
-3. Define a belongs_to association writer.
-4. Build a form field that will delegate to a belongs_to association writer. (post#category_name=) through controller mass assignment.
-5. Define a has_many association writer.
-6. Build a form field that will delegate to a has_many association writer. (owner#pet_names=) through controller mass assignment.
+2. Assign a form key based on a select box value directly through mass assignment (`post[category_id]`).
```
Comment (gj) [inline README.md]: @mendelB –– I guessed that 'FK' is an acronym for 'form key'. Is that wrong?

**Comment by gj** [comment]
Covers: https://github.com/learn-co-curriculum/forms-and-basic-associations-rails-lab/pull/7

**Comment by mendelB** [comment]
Hey @gj I think FK stands for 'foreign key' here (sending the value of the genre through params and setting the songs foreign key of genre_id). Mind just fixing that and I'll merge?

**Comment by gj** [comment]
Good call @mendelB. Done!

**Pull request merged**